docs/rawget-type-operator.md

```diff
@@ -0,0 +1,58 @@
+# `rawget` type operator
+
+## Summary
+
+This RFC proposes the addition on one type operator, `rawget`, which can be used to look up a specific property of another type *without* invoking the `__index` metamethod.
+
+## Motivation
+
+There exists an `index` type operator that allows developers to obtain a type of a property from classes / tables. If a type is not found in the given class / table, the operator recursively indexes into the `__index` metamethod to continue searching for the property. Sometimes, this could be an unwanted behavior. For example, given this code: 
+
+```luau
+local var1 = {
+  property = "hello"
+}
+
+local var2 = setmetatable({ }, { __index = seq })
```

Comment:
Is this meant to be `{ __index = var1}`?

Reply:
yes! thank you for pointing this out. my most recent commit has these mistakes patched